element_calcium_imaging/imaging.py

```diff
@@ -264,6 +265,15 @@ def make(self, key):
                 ProcessingTask * ProcessingParamSet * ProcessingMethod * scan.Scan & key
             ).fetch1("processing_method")
 
+            image_files = (
+                ProcessingTask * scan.Scan * scan.ScanInfo * scan.ScanInfo.ScanFile
+                & key
+            ).fetch("file_path")
```

Comment:
Do you need to join the other tables?
```suggestion
            image_files = (scan.ScanInfo.ScanFile & key).fetch("file_path")
```